demo_umami_content/src/InstallHelper.php

```diff
@@ -244,6 +245,40 @@ public function importPages() {
     return $this;
   }
 
+  /**
+   * Imports block contents.
+   *
+   * @return $this
+   */
+  public function importBlockContent() {
+    $block_content_metadata = [
+      'umami_recipes_banner' => [
+        'uuid' => '4c7d58a3-a45d-412d-9068-259c57e40541',
+        'info' => 'Umami Recipes Banner',
+        'type' => 'banner_block',
+        'field_title' => [
+          'value' => 'Baked Camembert with garlic, calvados and salami',
+        ],
+        'field_content_link' => [
+          'target_id' => 10,
```

Comment:
I tried this, and this linked to the test page rather than the camembert recipe page.

I guess there's no way of knowing the node ID for this specific recipe as the amount of migrated content could change.

Maybe we should look up the node ID based on title (and content type)?

Reply:
```php
$nodes  = $this->entityTypeManager->getStorage('node')->loadByProperties(['name' => $node_title]);
$node = reset($nodes);
```
should set you straight

Reply:
I will add it.